bin/conjur-cli.rb

```diff
@@ -9,7 +9,7 @@
 # Attempt to connect to the database.
 def connect
   require 'sequel'
-  
+
```

Comment:
Please try to configure your editor so that it only fixes whitespace on modified lines. This makes for a lot less noisy diffs. Thanks!